.github/workflows/test.yaml

```
@@ -1,8 +1,19 @@
-name: Python Test Suite
+name: CSL CI Test Suite
 on:
 - push
 - pull_request
 jobs:
+    name: prettier
```

you are missing a key

```suggestion
jobs:
  prettier:
    name: prettier
```